xclim/indices/_conversion.py

```diff
@@ -917,12 +918,26 @@ def snow_depth_from_amount(
         If snow water equivalent (`swe` [m]) is provided instead, will be converted to `snw` before calculating.
     snr : xr.DataArray, optional
         Snow density [kg/m^3].
+    const: Quantified, optional
+        Constant snow density [kg/m^3]
+        `const` is only used if `snr` is None.
 
     Returns
     -------
     xr.DataArray, [m]
         Snow depth.
+
+    Notes
+    -----
+    The estimated mean snow density value of 312 kg m-3 is taken from :cite:p:`sturm_swe_2010`.
```

Comment:
```suggestion
    The estimated mean snow density value of 312 kg m-3 is taken from :cite:t:`sturm_swe_2010`.
```

Reply:
`:cite:p:` provides parentheses citations, ie: `[Sturm, 2010]`, whereas `:cite:t:` provides in-text citations, ie: `Sturm [2010]`.